scripts/mysql_system_users.sql

```diff
@@ -100,19 +101,18 @@ GRANT ALTER, CREATE, SELECT, INSERT ON PERCONA_SCHEMA.xtrabackup_history TO 'mys
 
 -- Common. WITH GRANT OPTION is needed by clone-sst script
 GRANT BACKUP_ADMIN, EXECUTE ON *.* TO 'mysql.pxc.sst.role'@localhost WITH GRANT OPTION;
+-- For some reason this is also needed, although the docs say BACKUP_ADMIN is enough
```

Comment:
The reason why we need `SELECT` on `performance_schema.*` is because both the SST script and PXB need to query keyring status and log_status PFS tables respectively. 

Perhaps it is better we add a comment saying why we need it.

Reply:
Yup, it is good to have a real explanation rather than "for some reason" :).
Added.